src/page/UpdateForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { reduxForm } from 'redux-form';
+
+import { CreateForm } from './CreateForm.js';
```

Comment:
.js is needed ?

Reply:
[here](https://github.com/opstalent/redux-crud/pull/18/files#diff-d8d6d6933ff93f5a4ca3064abb323ad5R43)

Reply:
it is kind of a pattern I'm using here to avoid disambiguation, because there are also `md` files

Reply:
it is not needed but it is used in all files across lib

Reply:
ok